media/css/firefox/releasenotes.scss

```
@@ -31,11 +31,16 @@ $image-path: '/media/protocol/img';
         }
     }
 
-    .download-button {
+    blockquote + .download-button {
         @media #{$mq-md} {
             float: right;
         }
     }
+
+    .mzp-c-call-out-compact {
+        background-color: #fff;
```

nit: token for white.
```suggestion
        background-color: $color-white;
```